_includes/footer-menu.html

```diff
@@ -2,7 +2,7 @@
   <div class="row">
     <div class="col-sm-6 col-sm-offset-3">
       <ul class="footer-secondary-nav nav nav-pills nav-justified">
-        {{ site.data.footer-menu|build }}
+        {{ site.data.main-menu|build }}
```

Comment:
What happens if main-menu.yml has a second level of menu items, like uW does? This probably should still be generated with a footer-menu.yml file, as uW is, as the footer won't have the sub menu items. Right now the main menu is flat (one level) but in the future may get second level entries, which may screw up the footer. 

(compare https://github.com/unfoldingWord/unfoldingWord.github.io/blob/develop/_data/footer-menu.yml with https://github.com/unfoldingWord/unfoldingWord.github.io/blob/develop/_data/main-menu.yml)